docker-compose.yml

```diff
@@ -28,3 +33,54 @@ services:
       RABBITMQ_DEFAULT_VHOST: /libretime
       RABBITMQ_DEFAULT_USER: libretime
       RABBITMQ_DEFAULT_PASS: libretime
+
+  playout:
+    build: ./playout
+    volumes:
+      - ./docker/config.yml:/etc/libretime/config.yml:ro
+
+  liquidsoap:
+    build: ./playout
+    command: /usr/local/bin/libretime-liquidsoap
+    ports:
+      - "1234:1234"
+    volumes:
+      - ./docker/config.yml:/etc/libretime/config.yml:ro
```

Comment:
Wonder if we should save the liquidsoap generated configuration file to the host.